source/MaterialXTest/ShaderValid.cpp

```diff
@@ -431,15 +452,17 @@ static void runOSLValidation(const std::string& shaderName, mx::TypedElementPtr
 {
     mx::SgOptions options;
 
+    bool skipRendering = false;
+    const MaterialX::StringSet blackList({ "modulo_vector3FA_out", "modulo_color3FA_out" });
```

Comment:
Why don't you just fixing these broken tests instead? I thought you knew how to.

Reply:
Sure, I can change the implementation being called to be specific ones which pack the float to vector3 and color3 respectively. It doesn't fix the tests however, it just side-steps the testrender issue.